src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/IntentionalRerouteSpec.groovy

```diff
@@ -101,21 +107,21 @@ class IntentionalRerouteSpec extends HealthCheckSpecification {
         }
 
         and: "Init a reroute of the flow"
-        def rerouteResponse = northboundV2.rerouteFlow(flow.flowId)
+        def rerouteResponse = flow.reroute()
 
-        Wrappers.wait(WAIT_OFFSET) { assert northboundV2.getFlowStatus(flow.flowId).status == FlowState.UP }
+        Wrappers.wait(WAIT_OFFSET) { assert flow.retrieveFlowStatus().status == FlowState.UP }
 
         then: "The flow is successfully rerouted and goes through the preferable path"
-        def pathDto = northbound.getFlowPath(flow.flowId)
-        def newPath = PathHelper.convert(pathDto)
-        def newPathNodesV2 = PathHelper.convertToNodesV2(pathDto)
+        def newPathEntities = flow.retrieveAllEntityPaths()
+        def newPath = newPathEntities.getPathNodes()
+        def newPathNodesV2 = PathHelper.convertToNodesV2(newPath)
```

Comment:
can be replaced by
`newPathEntities.flowPath.path.forward.nodes.toPathNodeV2()` and in the tests below if it is applicable.

Reply:
Done